### DISCUSSION TITLE
Preparing for the first Release Candidate Version!

### DISCUSSION THREAD

**Discussion by zhenrong-wang** [zhenrong-wang/hpc-now#35] [Announcements]
Hello HPC-NOW folks,

After days of update, we are preparing for the first Release Candidate (RC1) of HPC-NOW. 

We've done so many works to make this project better in functionality, robustness, security, and compatibility. Including:

- **Security:** All of the sensitive files are well encrypted with AES, and the key comes from SHA-256 instead of the deprecated unsafe MD5.
- **Functionality:** HPC-NOW supports 7 public clouds, including Amazon Web Services, Microsoft Azure, Google Cloud Platform, Alibaba Cloud, Tencent Cloud, Huawei Cloud, Baidu BCE Cloud. It provides comprehensive functions for you to handle data transfer, HPC application management, as well as HPC cluster user management.
- **Robustness:** We enhanced many general functions to guarantee memory safety and avoid undefined behaviours.
- **Compatibility:** The HPC-NOW supports several X86_64 Operating Systems, including Microsoft Windows, macOS, GNU/Linux Distros (validated on openSUSE, Debian. Ubuntu, ArchLinux, CentOS/CentOS Stream, Rocky, and many more)

With the efforts above and the tests done, the HPC-NOW is much closer to productivity-ready. 

Please check the latest minor version (0.3.1.0127) and feel free to feedback to us.

HPC-NOW - make HPC easy, scalable, flexible, and powerful NOW.

**Comment by fuzzydd** [comment]
Great! I am testing the 0.3.1.0127 now.

**Comment by ngglov** [comment]
Good!

**Comment by l-lan0** [comment]
I still think the application manager is not easy to use. Especially when I'd like to build and use my own Application Repository.

**Reply by zhenrong-wang** [reply]
Yes, that needs to be better designed.
